selfdrive/car/hyundai/hyundaican.py

```diff
@@ -67,21 +67,10 @@ def create_clu11(packer, frame, clu11, button):
   return packer.make_can_msg("CLU11", 0, values)
 
 
-def create_lfa_mfa(packer, frame, enabled):
-  values = {
-    "ACTIVE": enabled,
-  }
-
-  # ACTIVE 1 = Green steering wheel icon
-
-  # LFA_USM 2 & 3 = LFA cancelled, fast loud beeping
-  # LFA_USM 0 & 1 = No mesage
-
-  # LFA_SysWarning 1 = "Switching to HDA", short beep
-  # LFA_SysWarning 2 = "Switching to Smart Cruise control", short beep
-  # LFA_SysWarning 3 =  LFA error
-
-  # ACTIVE2: nothing
-  # HDA_USM: nothing
-
+def create_lfahda_mfc(packer, enabled, lfahda_mfc, hda_active=0, hda_set_speed=0):
+  values = lfahda_mfc
```

Comment:
remove this

Reply:
what do you mean? the goal is to copy fields like `HDA_USM` and `LFA_USM` being sent by camera so I copied what we do for `CLU11`

Reply:
I just meant that line is unnecessary. why not just use lfahda_mfc?

Reply:
I made it consistent with both `create_clu11` and `create_lkas11` do you want me to change it and make it different?

If we want to update these I really think we should change them to not modify the param passed in because you would never expect these functions to modify the parameters (which can lead to bugs in the future).  I think that it is best to treat params as immutable unless it is obvious (e.g. `memcpy`) or special cases (e.g. not wanting to make a copy for super high performance)

Reply:
I'm not a fan of the pattern we have right now of copying in car state then modifying in car controller. One of the problems is it's not clear what fields are copied, and it relies on the can parser using a dict. I think it would be cleaner to just make a new dict in these car controller functions and explicitly copy out every field you want.

Reply:
that is similar to how it started, for example:
https://github.com/commaai/openpilot/blob/6e388948e3ecbb6b9ffa7a05fcd6ce3966ac5f27/selfdrive/car/hyundai/hyundaican.py#L47-L63

so what do you want to do for this PR?

Reply:
no longer relevant (not copying values from LFAHDA_MFC)